### DISCUSSION TITLE
Multiple servers

### DISCUSSION THREAD

**Discussion by spruce0** [plan-player-analytics/Plan#3022] [Q&A]
In the screenshots on the spigot page. It shows "Server 1" on one of the pages. If I installed this plugin on my Waterfall proxy, would it collect data from both of my servers behind it?

**Comment by AuroraLS3** [comment]
Multi proxy support is implemented in the dev newest dev release

**Reply by AuroraLS3** [reply]
https://github.com/plan-player-analytics/Plan/releases/tag/5.5.2379
